src/app/article/article.component.scss

```diff
@@ -104,22 +104,29 @@
 
 /* For mobile phones: */
 @media only screen and (max-width: 600px) {
```

Comment:
Use bootstrap 4 mixins  @include media-breakpoint-down(sm) instead of vanilla media queries